plugins/services/src/js/components/modals/ServiceActionDisabledModal.js

```diff
@@ -363,7 +363,7 @@ class ServiceActionDisabledModal extends React.Component {
   getGroupDeleteMessage() {
     const { actionID, service, intl } = this.props;
     const serviceID = service ? service.getId() : "";
-    const { selectedServices } = this.getServiceTypes(isPackage);
+    const { selectedServices } = this.getServiceTypes(isSDKService);
```

Comment:
Thanks for catching this @mlunoe. @orlandohohmeier @nLight - can you please check this before we put up PRs to DC/OS?